datalad/support/network.py

```diff
@@ -282,11 +283,24 @@ def __call__(self, url):
 
 def _guess_ri_cls(ri):
     """Factory function which would determine which type of a ri a provided string is"""
+    TYPES = {
+        'url': URL,
+        'ssh':  SSHRI,
+        'file': PathRI,
+        'datalad': DataLadRI
+    }
+    # go in exotic mode if this is an absolute windows path
+    win_split = win_splitdrive(ri)
+    # we need a drive and a path, otherwise this could be a false positive
+    if win_split[0] and win_split[1]:
+        # OMG we got something from windows
+        lgr.log(5, "Detected file ri")
+        return TYPES['file']
```

Comment:
not necessarily from windows -- could be some local masocist ;-)
```
hopa:~
$> touch c:\\bugi\dugi
2 19687.....................................:Tue 13 Jun 2017 04:53:35 PM EDT:.
hopa:~
$> ls -ld c:\\bugi\dugi
-rw------- 1 yoh yoh 0 Jun 13 16:53 c:\\bugidugi
```
but the conclusion holds ;)